test/unit/test_diskfile.py

```diff
@@ -34,7 +34,6 @@
 from . import utils
 from .utils import make_client_collection
 
-
```

Comment:
I believe this empty line is required for `flake` or some other checker.

Reply:
No precise reason for the deletion of this empty line, I guess it is a leftover from a previous edit. Fixed now